docs/long-john/bom/sourcing_and_print_guide.md

```diff
@@ -17,17 +19,17 @@ Currently there are no approved kits available, but fear not - the toolsetter us
 | Cables        | JST Connector 2 Position | JST-XH             | 1   |
 
 !!! info annotate "Return Spring"
-    The Long John Toolsetter can be fitted with a compression spring to help the contact return. You may choose to fit this, however it isn't required. Any appropriately sized low force compression spring should be suitable.
+    The Long John Toolsetter can be fitted with a compression spring to help the contact return. You may choose to fit this, however it isn't required. Any appropriately sized low-force compression spring should be suitable.
 
 # Printing Guide
 
 The toolsetter part has a sacrificial floor to allow unsupported features to be printed successfully, and overhangs that will become more difficult to print with thicker layers. These floors are designed to work at a layer height of **0.2mm**, so they may not be sliced correctly if your layer height is not 0.2mm.
 
 **Unless you're willing to experiment and almost certainly waste filament, stick with 0.2mm layers.**
 
-We provide a **Minimum Wall Thickness** for every printable part, which is the minimum **total** thickness of your perimeter lines and top and bottom solid layers. This depends on the size of your nozzle and line width settings in your slicer so must be calculated.
+We provide a **Minimum Wall Thickness** for every printable part, that is the minimum **total** thickness of your perimeter lines and top and bottom solid layers. This depends on the size of your nozzle and line width settings in your slicer so it must be calculated.
```

Comment:
`which` vs `that` is an Americanisation. 

In Britbong English we use `which` as a nonrestrictive clause, i.e. where it adds additional information to the sentence that has no bearing on the identification of the subject (in this case, the minimum wall thickness).

My understanding is that American English only ever uses `that`.

Happy to defer to the majority but the guides were written in British English and I'd prefer we stick with it where possible.

I would _prefer_ less changes over more that